zap/src/main/java/org/parosproxy/paros/model/Session.java

```diff
@@ -1639,7 +1642,9 @@ public ParameterParser getFormParamParser(String url) {
      * @param uri
      * @return
      * @throws URIException
+     * @deprecated use #getUrlParameters(String)
      */
+    @Deprecated
```

Comment:
Should these annotations have a `TODO Add version` ?

Reply:
Added in the comment